native/cocos/renderer/pipeline/forward/ForwardStage.cpp

```diff
@@ -171,6 +171,17 @@ void ForwardStage::render(scene::Camera *camera) {
         }
         _clearColors[0].w = camera->getClearColor().w;
         // color
+        framegraph::RenderTargetAttachment::Descriptor colorAttachmentInfo;
```

Comment:
If the AR background pass can be merged into the forward color pass, no special processing is required here. It is also possible to draw the background behind opaque objects to enable early-z test.